framework/packages/abstract-client/tests/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod app;
```

Comment:
I did not see this, will use it instead :)

Reply:
A quick attempt to use it runs into the issue that the mock app does not call `abstract_app::cw_orch_interface!` so some extra work would be necessary there. Any reason why we can't refactor the mock app to use this interface? I think it would more closely resemble the customer experience which is what we should aim to do with tests.